kuma/static/styles/zone-addons.scss

```diff
@@ -0,0 +1,34 @@
+@import 'includes/vars';
+@import 'includes/mixins';
+
+/*
+Addons / Web Extentions zone
+********************************************************************** */
+
+
+/*
+imports
+====================================================================== */
+@import 'zones/components/webext-summary-compat-table';
+
+
+/* =================================================================== */
+
+.addon-sdk-api-name {
+    background-color: rgba(212, 221, 228, .15);
+    padding: 3px;
+}
```

Comment:
To delete them in a follow up PR once we've switched their zones to include the individual zone style sheets.